app/components/wallet/AccountsOverview.js

```diff
@@ -9,6 +9,8 @@ import { copyBlack, copyWhite } from '/assets/images';
 import { getAbbreviatedText, getAddress, formatSmidge } from '/infra/utils';
 import { smColors } from '/vars';
 import type { Account, Action } from '/types';
+import type { RouterHistory } from 'react-router-dom';
+import Link from '../../basicComponents/Link';
```

Comment:
we have absolute path definition in webpack, no need in '../../'

Reply:
fix